.github/scripts/filter_test_configs.py

```diff
@@ -155,19 +131,25 @@ def get_labels(pr_number: int) -> Set[str]:
     }
 
 
+def has_labels(labels: Set[str], label_regex: Any) -> Set[str]:
+    """
+    Return true if there is at least one label matching the regex
```

Comment:
nit: `filter_labels` or `get_labels` might be better names for this function, also the doc string isn't right